packages/flame/lib/src/components/input/toggle_button_component.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flame/components.dart';
+import 'package:flame/events.dart';
+import 'package:flutter/foundation.dart';
+
+class ToggleButtonComponent extends AdvancedButtonComponent {
```

Comment:
Add some short dartdocs here (and on the other component)

Reply:
Ready! Correct my English :) I promise to expand the dartdocs later - but I still need to add labels.